src/bower.json

```diff
@@ -6,19 +6,12 @@
   "dependencies": {
     "jquery": "2.0.*",
     "underscore": "~1.6.0",
-    "bootstrap": "~3.1.1",
```

Comment:
I think Tomas made these changes ages ago and never merged to `master`. Ignore them.